### PR TITLE
moved some files around, created the main review "button" component

### PR DESCRIPTION
**Pull request by hongmm**
A lot of changes in regards to file organization + my frontend. I only really need eyes on the index.jsx and review.jsx components

### CODE REVIEW

**Comment by hongmm** [comment]
image

File: client/src/components/Review.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  const getReviewAverage = () => {
+    if (reviews.length === 0) {
+      return 0;
+    }
+    console.log(reviews);
+    let sum = 0;
+    reviews.forEach((review) => { sum += review.score; });
+    return sum / reviews.length;
+  };
+  // getReviewAverage();
```
Comment (Ahmed-Elawad) [inline client/src/components/Review.jsx:16]: @hongmm, don't forget your commented out code!

File: client/src/index.jsx [hunk 2]
```diff
@@ -23,15 +25,16 @@ const App = (props) => {
     getAllReviews();
   }, []);
 
+  const eventHandler = (event) => {
+    event.preventDefault();
+    const val = event.target.value;
+    console.log(val);
+  };
+
```
Comment (Ahmed-Elawad) [inline client/src/index.jsx:33]: @hongmm, your event handler here doesn't seem to do anything. I think in this case you'd leave it out until it has useful functionality.

File: client/src/index.jsx [hunk 3]
```diff
@@ -23,15 +25,16 @@ const App = (props) => {
...
     <div>
-      Reviews:
-      <div>
-        Rating stars
-      </div>
-      <div>
-        Rating count
-      </div>
+      <Review reviews={reviews} eventHandler={eventHandler} />
+      {/* <Modal /> */}
```
Comment (Ahmed-Elawad) [inline client/src/index.jsx:37]: @hongmm, don't forget your commented out code!

File: index.js [hunk 4]
```diff
@@ -2,19 +2,16 @@ const express = require('express');
...
-app.use(express.static(path.join(__dirname, '/../client/dist')));
+app.use(express.static(path.join(__dirname, 'client', 'dist')));
 
-// sequelize.query('SELECT * FROM reviews LIMIT 5', { type: Sequelize.QueryTypes.SELECT }
-app.get('/:productId', (req, res) => sequelize.query(`SELECT * FROM reviews WHERE product_id = ${req.param('productId')}`, { type: Sequelize.QueryTypes.SELECT })
-  .then((result) => res.sendFile(path.join(__dirname, '/../client/dist/index.html')))
-  .catch((error) => res.send(error)));
+app.get('/:productId', (req, res) => res.sendFile(path.join(__dirname, 'client', 'dist', 'index.html')));
 
 app.get('/api/reviews/:productId', (req, res) => sequelize.query(`SELECT * FROM reviews WHERE product_id = ${req.param('productId')}`, { type: Sequelize.QueryTypes.SELECT })
```
Comment (Ahmed-Elawad) [inline index.js:16]: @hongmm, This app.get callback seems to flow into another line. I'd move this down into its own block scope.

**Review by Ahmed-Elawad — COMMENTED** [review]
@hongmm, changes look good Matt. Left a few comments on styling for you.

**Pull request merged**